### PR TITLE
fuzzers: Add 000-init-db fuzzer.

### LINKED ISSUE #604: 060-bram-cascades missing dependency?
```
060-bram-cascades        -    0s: Starting @ 2019-02-03T14:56:03.409587
060-bram-cascades        -    0s: Running make -C /tmpfs/src/github/symbiflow-prjxray-continuous-db-artix7/fuzzers/060-bram-cascades run (with MAKEFLAGS=' -j --jobserver-fds=3,4 -- MAX_VIVADO_PROCESS=96')
060-bram-cascades        -   10s: Still running (1m:124.81%, 5m:150.46%, 15m:85.99%).
060-bram-cascades        -   10s: make---sh---bash---make---sh---vivado.sh---vivado---loader---vivado---2*[{vivado}]
060-bram-cascades        -   16s: Finishing (User:15s System:1s).
060-bram-cascades        -   16s: Failed @ 2019-02-03T14:56:19.990161 with exit code: 2
060-bram-cascades        -   16s: --------------------------------------------------------------------------
060-bram-cascades        -   16s: + MAKE=make
060-bram-cascades        -   16s: + echo make
060-bram-cascades        -   16s: + i=1
060-bram-cascades        -   16s: + true
060-bram-cascades        -   16s: + make ITER=1 cleaniter
060-bram-cascades        -   16s: + make ITER=1 build/todo.txt
060-bram-cascades        -   16s: cp: cannot stat '/tmpfs/src/github/symbiflow-prjxray-continuous-db-artix7/database/artix7/segbits*.db': No such file or directory
060-bram-cascades        -   16s: make[2]: *** [build/database/seeded] Error 1
060-bram-cascades        -   16s: make[2]: *** Waiting for unfinished jobs....
060-bram-cascades        -   16s: make[1]: *** [run] Error 2
060-bram-cascades        -   16s: 
060-bram-cascades        -   16s: --------------------------------------------------------------------------
060-bram-cascades        -   16s: Failed @ 2019-02-03T14:56:19.990161 with exit code: 2
```

### PR DESCRIPTION
**Pull request by mithro**
Creates the .db files via make rather than via sourcing settings.sh
allowing pip fuzzers to interact correctly with `make clean`.

Fixes #604.

Signed-off-by: Tim 'mithro' Ansell <me@mith.ro>

### CODE REVIEW

File: settings/artix7.sh [hunk 1]
```diff
@@ -24,4 +24,3 @@ export XRAY_PIN_05="G22"
 export XRAY_PIN_06="F21"
 
 source $(dirname ${BASH_SOURCE[0]})/../utils/environment.sh
-init_db
```
Comment (kgugala) [inline settings/artix7.sh]: this should be also removed from zynq and kintex settings. I think we call this function in some minitests (ROI harness)

Comment (mithro) [reply settings/artix7.sh]: Done.

I couldn't find `init_db` in any of the minitest directories?

File: fuzzers/000-init-db/Makefile [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      clbll \
+      clblm \
+      dsp \
+      hclk \
+      int \
+
+DB_SIMPLE=$(addprefix mask_,$(MASKS)) $(addprefix ppips_,$(PPIPS)) $(addprefix segbits_,$(SEGBITS))
+
+# Extra (segbits|mask)_bram_(l|r).block_ram.db files
+# FIXME: Are these things also needed for the DSP blocks?
```
Comment (litghost) [inline fuzzers/000-init-db/Makefile]: Not to our knowledge

File: settings/artix7.sh (hunk 1 above)
Comment (kgugala) [reply settings/artix7.sh]: I guess we didn't use it there.

**Pull request merged**